qutip/solver/brmesolve.py

```diff
@@ -25,10 +26,11 @@ def brmesolve(
     psi0: Qobj,
     tlist: ArrayLike,
     a_ops: list[tuple[QobjEvoLike, CoefficientLike]] = None,
-    e_ops: EopsLike | list[EopsLike] | dict[Any, EopsLike] = None,
+    sec_cutoff: float = 0.1,
+    *pos_args,
```

Comment:
Could we add `pos_args` to the argument list and use it to document that the parameters following it are now keyword only and what the previous ordering was.

I would also be happy for the order to be finally changed as soon as possible -- e.g. 5.2. Having this clumsy signature is probably more detrimental than small breakages.

Reply:
It depend on the pace of releases...
We should not have breaking changes in minor release without deprecation warning. They usually come with a 1 year head up in other projects. 2 minor versions in a year feels a reasonable pace, but we don't have set schedule making it difficult...